styles.css

```diff
@@ -31,6 +31,110 @@ body {
   padding: 10px;
 }
 
+/* mobile menu */
+
+.mobile-menu {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 27px;
+  height: 16px;
+  cursor: pointer;
+}
+
+.bar {
+  background-color: blue;
+  height: 1.9px;
+}
+
+.mobile-menu.active .bar:nth-child(2) {
+  opacity: 0;
+  transition: 0.3s ease;
+  background: blue;
+}
+
+.mobile-menu.active .bar:nth-child(1) {
+  transform: translateY(8px) rotate(45deg);
+  background: whitesmoke;
+  transition: 0.2s ease;
+  height: 4px;
+}
+
+.mobile-menu.active .bar:nth-child(3) {
+  transform: translateY(-8px) rotate(-45deg);
+  background: white;
+  transition: 0.6s ease;
+  height: 4px;
+}
+
+.nav_icon-two {
+  list-style-type: none;
+  left: -120%;
+  height: 100%;
+  top: 0;
+  flex-direction: column;
+  position: fixed;
+  backdrop-filter: blur(8px);
+  background: #6070ff;
+  width: 100%;
+  transition: 0.5s ease;
+  mix-blend-mode: multiply;
+  padding-top: 150px;
+}
+
+.nav_icon-two li {
+  padding: 2rem 0 0 10rem;
+}
+
+.resume a {
+  text-decoration: none;
+  color: #396df2;
+}
+
+.nav_icon-two .nav-link {
+  color: white;
+  font-size: 25px;
+  text-decoration: none;
+  font-weight: 100;
+}
+
+.logo a {
+  font-size: 18px;
+  text-decoration: none;
+  color: blue;
+}
+
+.connect a {
+  text-decoration: none;
+  color: #7f8cff;
+}
+
+.action a {
+  color: #396df2;
+  text-decoration: none;
+  border-radius: 8px;
+}
+
+.upper-part {
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.about-connect a {
+  text-decoration: none;
+  color: #7f8cff;
+}
+
+.nav_icon-two .nav-link a {
+  color: white;
+  font-size: 100px;
+  text-decoration: none;
+}
+
+.nav_icon-two.active {
+  left: 0;
+}
+
```

Comment:
Good job on completing the mobile menu it's working perfectly, there is only one thing to mention, it's about the design, please know that you should follow the Figma template design, your design is looking like this:
  ![image](https://user-images.githubusercontent.com/20567503/196421005-89e866fc-0b84-4351-afdc-c65bb32b2ee4.png)
  And the Figma design is looking like this:
  ![image](https://user-images.githubusercontent.com/20567503/196421172-41aef278-25a6-4a20-bc3d-1a5989582de8.png)
  From what I see the following should be changed:
  - [ ] The text is leaning to the right, it should be on the left according to the design, kindly change that.
  - [ ] The text isn't bold, and the size isn't the same as the one in Figma design, kindly adjust that as well.
  - [ ] The padding you put on top is a lot bigger than what is show on the Figma design, kindly adjust that as well.